R/load_fonts.R

```diff
@@ -8,23 +8,24 @@
 #' @export
 #'
 load_arcadia_fonts <- function(custom_font = "Suisse", fallback_font = "sans") {
-  showtext::showtext_auto(enable = TRUE)
+  # define font names to check for
+  font_names <- c("Suisse Int'l", "Suisse Int'l Semi Bold", "Suisse Int'l Medium", "Suisse Int'l Mono")
```

Comment:
Should this use / import the constants from `R/arcadia.R`?

Reply:
I'm not sure how to export constants across scripts, so for now I'll leave it like this and make an issue. Unless I have to make a specific function like `define_constants()` somewhere and export it so that it's available to all other scripts.